x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -109,3 +121,6 @@ export const getTheme = () => {
   const defaultTheme = isDarkMode ? DARK_THEME : LIGHT_THEME;
   return mergeWithDefaultTheme(theme, defaultTheme);
 };
+
+const kibanaTimezone = chrome.getUiSettingsClient().get('dateFormat:tz');
```

Comment:
I know this has already been merged, but just a heads up that you can use a `useContext` hook with the `KibanaConfigContext` ala [here](https://github.com/elastic/kibana/blob/8032f82b18c88514d2ba675d434a57d1d081c4e9/x-pack/plugins/siem/public/components/formatted_date/index.tsx#L21) to access `dateFormatTz` or any other Kibana config setting as well.